pymc3/sampling.py

```diff
@@ -1508,6 +1508,14 @@ def _mp_sample(
 
 
 def _choose_chains(traces, tune):
+    """
+    Pick traces such that the effective sample size is maximised.
```

Comment:
The choice of words is misleading: "effective sample size" (ESS) is a diagnostic and this code has nothing to do with ESS.

Should be something like "Filter and slice traces such that (n_traces * len(shortest_trace)) is maximized.

Reply:
oops, thanks!